### DISCUSSION TITLE
Static IP in nextjs API

### DISCUSSION THREAD

**Discussion by Tushar0704** [vercel/next.js#44356] [Help]
It's my first time using next api and I am trying to use a 3rd party API where I need to whitelist the IP of the server from where the request will be coming through. I know how to do this in AWS lambda's using elastic IPs and VPC, but I have no idea how this can be implemented in next itself. Does anyone have any idea if this is currently possible or not?

**Comment by GabenGar** [comment]
Define "whitelist the IP" and in what context do you need to access it?

**Comment by ariabov** [comment]
Check out [Fixie for Vercel](https://vercel.com/integrations/fixie) as I think it does exactly what you need.
